README.md

```diff
@@ -9,9 +9,31 @@ Welcome to Spokestack Python! This library is intended for developing [voice int
 
 ## Get Started
 
-### System Dependencies
+### Installation with pip
+
+Once system dependencies have been satisfied, you can install the library with the following.
+
+```shell
+pip install spokestack
+```
+
+### Install Tensorflow
 
-There are some system dependencies that need to be downloaded in order to install spokestack via pip.
+This library requires a way to run [TFLite](https://www.tensorflow.org/lite) models. There are two ways to add this ability. The first is installing the full [Tensorflow](https://www.tensorflow.org/) library.
+
+The full Tensorflow package is installed with the following:
+
+```shell
+pip install tensorflow
+```
+
+#### TFLite Interpreter (Embedded Devices)
+
+In use cases where you require a small footprint, such as on a Raspberry Pi or similar embedded devices, you will want to install the TFLite Interpreter. You can install it for your platform by [following the instructions](https://www.tensorflow.org/lite/guide/python#install_just_the_tensorflow_lite_interpreter).
```

Comment:
Why not include the `pip install` command they list, just for the convenience of not clicking away (and to match the previous section)?

Reply:
yep. thanks for pointing that out again. might as well get it with this one also.